marquee/formatter.py

```diff
@@ -2,7 +2,6 @@
 import json
 from logging import Formatter
 
-
```

Comment:
This line you removed still belongs. Prefer 2 lines between top level functions and classes. Just linting stuff.